client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-table-expand/site-table-popover.tsx

```diff
@@ -0,0 +1,47 @@
+import { Button, Popover } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import { forwardRef } from 'react';
+import { useJetpackAgencyDashboardRecordTrackEvent } from '../../hooks';
+
+import './style.scss';
+
+const SiteTablePopover = (
+	{ savePreferenceType }: { savePreferenceType: () => void },
+	ref: any
+) => {
+	const translate = useTranslate();
+
+	const recordEvent = useJetpackAgencyDashboardRecordTrackEvent( null, true );
+
+	const handlePopoverDismiss = () => {
+		savePreferenceType();
+		recordEvent( 'expandable_block_popover_dismiss_click' );
+	};
+
+	const content = (
+		<div className="site-table__expand-popover-content">
+			<div className="site-table__expand-popover-header">{ translate( 'See more info' ) }</div>
+			<div className="site-table__expand-popover-body">
+				{ translate( 'Click here and access to more info about your site.' ) }
```

Comment:
Is this copy the correct one? I believe it has a typo, but we can make it more user-friendly: `Click here for more info about your site.`

Reply:
Yes, agree. Fixed here - https://github.com/Automattic/wp-calypso/pull/75386/commits/fc3f869de5ad5cd23d04d064be86711d6b75cfda